modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

```diff
@@ -56,10 +56,14 @@ To create an HTTP global element for the connector:
 +
 image::sap/sap-hana-http-listener.png[]
 +
-** *Protocol*: Protocol selected for the HTTP endpoint. This can be HTTP or HTTPS (secure).
-** *Host*: IP address where your Mule app listens for requests.
-** *Port*: Port address where your Mule app listens for requests.
-** *Base Path*: Path where your Mule app listens for requests.
+[%header,cols=30s,70a"]
+|===
+|Studio Field |Description
+|Protocol |Protocol selected for the HTTP endpoint. This can be HTTP or HTTPS (secure).
```

Comment:
Do we really need to modify HTTPS with "(secure)"? Even tech dummies like me understand that!  :-)